app/assets/stylesheets/application.scss

```diff
@@ -11,6 +12,14 @@
   margin-bottom: 2em;
 }
 
+fieldset {
+  border: 2px #111 solid;
```

Comment:
Color literals like `#111` should only be used in variable declarations; they should be referred to via variable everywhere else.